wrt/wrt-apptools-android-tests/README

```diff
@@ -0,0 +1,34 @@
+## Introduction
+
+This test suite is for wrt-apptools-android-tests
+
+## Precondition
+
+1. Connect Android devices to your localhost
+2. Need to edit the file "wrt-apptools-android-tests/arch.txt" according to the type of test device.
```

Comment:
x86 is default arch in arch.txt, so here you just introduce when the test device is arm arch, tester should update the arch.txt